### PR TITLE
Update documentation for security_monitoring_rule

### PR DESCRIPTION
**Pull request by ebarti**
Fixes https://github.com/DataDog/terraform-provider-datadog/issues/1394

### CODE REVIEW

File: docs/resources/security_monitoring_rule.md [hunk 1]
```diff
@@ -99,8 +99,8 @@ Optional:
 - **aggregation** (String) The aggregation type. Valid values are `count`, `cardinality`, `sum`, `max`, `new_value`.
 - **distinct_fields** (List of String) Field for which the cardinality is measured. Sent as an array.
 - **group_by_fields** (List of String) Fields to group by.
-- **metric** (String) The target field to aggregate over when using the sum or max aggregations.
-- **name** (String) Name of the query.
+- **metric** (String) The target field to aggregate over when using the `sum`, `max` or `new_value` aggregations.
```
Comment (kayayarai) [inline docs/resources/security_monitoring_rule.md:102]:
```suggestion
- **metric** (String) The target field to aggregate over when using the `sum`, `max`, or `new_value` aggregations.
```

**Review by kayayarai — APPROVED** [review]
Tiny edit, otherwise LGTM

**Comment by skarimo** [comment]
Hi, the docs are auto generated using tfplugindocs. Could you update the attribute desscription in the resource files and run `make docs` https://github.com/DataDog/terraform-provider-datadog/blob/master/datadog/resource_datadog_security_monitoring_rule.go#L225

Thanks!

**Review by adussarps — APPROVED** [review]
👍 This looks alright.

**Comment by skarimo** [comment]
Closing in favor of: https://github.com/DataDog/terraform-provider-datadog/pull/1447

**Pull request closed without merging**